### ISSUE TITLE
Проверка

### ISSUE THREAD

**Issue by accetone** [SanychBY/oop_lab1#1]
Проверю позже

**Comment by accetone** [comment]
Не прошло компиляцию
![clip2net_150907195249](https://cloud.githubusercontent.com/assets/7840570/9720774/5dee4f60-559a-11e5-9815-21437b26b5a2.png)

**Comment by SanychBY** [comment]
Странно. Я скопировал код из main.cpp и все заработало. У меня VS 2015. Если не работает, то тогда покажу непосредственно на лабе.
![default](https://cloud.githubusercontent.com/assets/8833670/9720909/cf5d4f7e-559b-11e5-8081-112c43d9c324.png)

**Comment by accetone** [comment]
Да, в 2015 скомпилировалось, видимо возможности из последнего стандарта C++ используются для массивов. 
- - - -
1. Для каждого поля (день, месяц, год) установщик (set, сеттер) и получатель (get, геттер) значения должен быть написано отдельно, а не одним методом push. 
2. Нарушен принцип инкапсуляции, к полям есть непосредственный доступ т.к. они public
3. Нет конструктора копирования
4. https://github.com/SanychBY/oop_lab1/blob/master/main.cpp#L11
Ужасно, для каждого объекта будет создавать массив с константами, подумай как это улучшить

**Comment by accetone** [comment]
+ читаем code style (code conventions) приложение после первой лабы
